### PR TITLE
Remove file

### PR DESCRIPTION
**Pull request by akshithP**
## Proposed changes

-Added 'remove file' functionality
-Filenames are displayed as a list in a container
-Added some CSS code for the container & the remove button

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] DevOps related

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [project guidelines](https://github.com/Visual-Testing-for-Android-Apps/Project-guidelines) doc
- [x] I locally linted my code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments
After dragging the files into upload box
Before removing file

Before clicking on remove button for Pic_1 (Couldn't capture the mouse pointer whilst hovering over the 'remove button')
Before clicking on remove

After clicking on remove button for Pic_1
After removing the file

### CODE REVIEW

**Review by leanne-b — APPROVED** [review]
Looks good!

**Review by idomo1 — CHANGES_REQUESTED** [review]
👍 Nice work.
A couple requests, could we please try and centre the file name vertically?
Also, lucky you, you get to resolve some merge conflicts 😅

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "configurations": [
+    {
+      "type": "pwa-chrome",
+      "request": "launch",
+      "name": "Launch Chrome against localhost",
+      "url": "http://localhost:8080",
+      "webRoot": "${workspaceFolder}"
+    }
+  ]
+}
```
[the remark this answers is not in the document]
Comment (Akshith-Patil) [reply .vscode/launch.json]: Sorry about that, I think it was added when crystal pushed to the branch. I've deleted the file

File: src/components/mainPage/upload.css [hunk 2]
```diff
@@ -18,6 +19,16 @@
   text-align: center;
 }
 
+.remove-btn {
+  background-color: rgb(194, 10, 47);
+  border-radius: 0.4rem;
+  color: white;
+  font-size: 18px; 
+  cursor: pointer; 
+  padding: 4px 10px;
```
[the remark this answers is not in the document]
Comment (Akshith-Patil) [reply src/components/mainPage/upload.css]: Yep, converted the px to rem now.

**Comment by idomo1** [comment]
Thanks, just a couple more things.
image

Some issues with mobile display with a long filename. Also, not enough margin between the list and the next section (causing the white circle overlap in the bottom right).

Then need to resolve those merge conflicts

File: src/components/mainPage/upload.css [hunk 3]
```diff
@@ -29,3 +40,22 @@
...
+  border-radius: 0.1rem;
+  padding: 1rem 1rem;
+  text-indent: 1.3rem;
+  font-size: 1.125rem;
+  font-weight: 34.375rem;
+}
+
+.vertical-center {
+  position: relative;
+  transform: translateY(13%);
```
Comment (idomo1) [inline src/components/mainPage/upload.css:56]: This is a pretty strange way to centre the text 😆

File: src/components/mainPage/upload.css [hunk 4]
```diff
@@ -29,3 +40,22 @@
...
 }
+
+.file-container{
+  background-color: #efefef;
+  border: 0.1rem solid black; 
+  border-radius: 0.1rem;
+  padding: 1rem 1rem;
+  text-indent: 1.3rem;
+  font-size: 1.125rem;
+  font-weight: 34.375rem;
```
Comment (idomo1) [inline src/components/mainPage/upload.css:51]: font-weight doesn't need to use rem. font-weight is unitless https://www.w3.org/TR/CSS1/#font-weight

Comment (Akshith-Patil) [reply src/components/mainPage/upload.css]: Ok, thanks for the feedback. I've fixed the filenames issue, also added margin space at end of the list. Is there anything else I should modify before I start resolving the merge conflicts.

**Review by idomo1 — APPROVED** [review]
Great work 👍 
Found a weird bug, happy to leave fixing it for later.
- Add a file
- Add the same file
- Press "Remove" on the first file
- Bug: You get navigated to the results page

**Comment by idomo1** [comment]
FYI you should be able to fix the CSS linting errors by running `npx stylelint --fix ./src/components/mainPage/upload.css`

**Comment by Akshith-Patil** [comment]
That's strange, yea I'll try to fix it later. Thanks, ill fix up the lining errors rn.

**Pull request merged**